packages/calling-component-bindings/src/index.ts

```diff
@@ -4,14 +4,23 @@
 export type { CallingBaseSelectorProps } from './baseSelectors';
 export * from './callControlSelectors';
 export { createDefaultCallingHandlers } from './handlers/createHandlers';
+/* @conditional-compile-remove(teams-identity-support) */
+export { createDefaultTeamsCallingHandlers } from './handlers/createTeamsCallHandlers';
 export type { ParticipantListSelector } from './participantListSelector';
 export type { ParticipantsButtonSelector } from './participantsButtonSelector';
 export type { VideoGallerySelector } from './videoGallerySelector';
 export type { ErrorBarSelector } from './errorBarSelector';
 /* @conditional-compile-remove(PSTN-calls) */
 export type { HoldButtonSelector } from './callControlSelectors';
 
-export type { CallingHandlers } from './handlers/createHandlers';
+export type { CallHandlersOf, CallTypeOf } from './handlers/createHandlers';
+export type { CallingHandlers } from './handlers/createACSCallHandlers';
+/* @conditional-compile-remove(teams-identity-support) */
+export type { TeamsCallingHandlers } from './handlers/createTeamsCallHandlers';
+export type { CallingHandlersCommon } from './handlers/createHandlersCommon';
```

Comment:
Naming nit: To go with `CallingHandlers` and `TeamsCallingHandlers`, I'd name this `CommonCallingHandlers` (as `Teams` is a prefix, so should be `Common`).

Reply:
Also, should `CallingHandlersCommon` also be a beta-only export?

Reply:
CallingHandlersCommon would be used and required by some stable feature (imagine components will be asking for CallingHandlersCommon  instead of CallingHandlers)